ui/src/components/empty-state/empty-state.tsx

```diff
@@ -14,13 +14,13 @@ export const EmptyState = () => {
             : STRING.MESSAGE_NO_RESULTS
         )}
       </span>
-      {activeFilters.length && (
+      {activeFilters.length ? (
```

Comment:
@annavik I had to decide which of these changes to keep in the merge. Will you confirm that I chose the right one?

Reply:
Yes that looks right, thank you! Since `activeFilters.length` is a number, not a boolean, the later solution is better. Before we actually had a `0` mistakenly rendered, instead of nothing in the case of no filters.

If you want to go down the React rabbit hole further here is a blog post explaining this little thing: https://www.codemzy.com/blog/react-render-0-conditional

Reply:
If you see a `0` out of context in any web app, it's probably because of this, it's a common mistake which I sometimes still do myself!! Another solution could be to cast the number to boolean `!!activeFilters.length`, but I personally think the ternary makes code a bit more readable in this case.